tests/install-plugins-plugins-cli/Dockerfile

```
@@ -1,3 +1,3 @@
 FROM bats-jenkins
 
-RUN /usr/local/bin/install-plugins.sh junit:1.6 ant:1.3 mesos:0.13.0 git:latest filesystem_scm:experimental docker:1.1.6
+RUN & C:/ProgramData/Jenkins/jenkins-plugin-cli.ps1 --plugins junit:1.6 ant:1.3 mesos:0.13.0 git:latest filesystem_scm:experimental docker:1.1.6
```

I don't recognize the `&` character in this position.  I'm accustomed to it being used to run two commands on a single Windows command line, but don't recognize it at the start of the command.  Can you explain further?
```suggestion
RUN & C:/ProgramData/Jenkins/jenkins-plugin-cli.ps1 --plugins junit:1.6 ant:1.3 mesos:0.13.0 git:latest filesystem_scm:experimental docker:1.1.6
```

---

It's the "call" operator: https://ss64.com/ps/call.html

Here is the description from that page: 

```
Many times you can execute a command by just typing its name, but this will only run if the command is in the environment path. Also if the command (or the path) contains a space then this will fail. Surrounding a command with quotes will make PowerShell treat it as a string, so in addition to quotes, use the & call operator to force PowerShell to treat the string as a command to be executed.
```